### ISSUE TITLE
revert added --cert flag for http client or fix HTTPS server

### ISSUE THREAD

**Issue by bartlomieju** [denoland/deno#4247]
After #3972 landed I'm getting `cafile_info` failures 5/10 CI runs which make it unusable. 

https://github.com/denoland/deno/pull/4246/checks?check_run_id=484893859

We should either revert the PR or fix underlying server issue

**Comment by bartlomieju** [comment]
CC @geoFlux

**Comment by geoFlux** [comment]
I'm trying to look into this, but I'm getting a compile error now.

**Comment by ry** [comment]
I was looking into this a few days ago - I believe it’s something related to the cwd in the tests... I don’t think it’s some fundamental problem with —cert.

**Comment by Sharpiro** [comment]
I was able to confirm that by using `--cert` I was able to use Deno with remote modules while behind a corporate proxy for the first time.

**Comment by geoFlux** [comment]
This is a hard error to look into. I've tried to reproduce it on my local machine, but I can't get it to error out no matter how many times I run it. I'm even running it on a mac, which is what is showing as failing here.

**Comment by ry** [comment]
@geoFlux I will look into it today. I can reproduce it reliably.

**Comment by bartlomieju** [comment]
> @geoFlux Thanks - I will look into it today. I can reproduce it reliably.

@ry please write down how to reproduce it, I'll see to a fix

**Comment by ry** [comment]
I don't see failures with cafile_info, I see them with cafile_install_remote_module (which was disabled).

I've tried to re-enable it here https://github.com/denoland/deno/pull/4429

**Comment by bartlomieju** [comment]
Closing in favor of #4512
